src/app/src/test/kotlin/org/imperial/mrc/hint/integration/ExceptionHandlerTests.kt

```diff
@@ -138,10 +133,10 @@ class ExceptionHandlerTests : SecureIntegrationTests()
     }
 
     @Test
-    fun `does not include original message from arbitrary exceptions`()
+    fun `does not include original message from handle exceptions`()
     {
         val sut = HintExceptionHandler(RandomErrorCodeGenerator(), ConfiguredAppProperties())
-        val result = sut.handleArbitraryException(PSQLException("some message", mock()), mock())
+        val result = sut.handleException(PSQLException("some message", mock()), mock())
```

Comment:
So this is also a behaviour change that I'm not sure whether we want - previously we were returning known SpringBoot error messages as "trace" objects which I think has been useful for debugging...